src/payments/daily-refill/notify.ts

```
@@ -0,0 +1,142 @@
+  nextRefreshAt.setUTCHours(0, 0, 0, 0);
+  nextRefreshAt.setUTCDate(nextRefreshAt.getUTCDate() + 1);
+
+  const apns = createApnsService();
+  const fcm = createFcmService();
+
+  await Promise.all(
+    [...devicesByAccount.entries()].flatMap(([accountId, accountDevices]) => {
+      const entry = refilledByAccount.get(accountId);
+      if (!entry) return [];
+
+      const payload: CreditsRefilledPayload = {
```

ESLint

[error] Insert `··`

(prettier/prettier)